cmd/template/cluster/provider/azure.go

```diff
@@ -85,6 +85,18 @@ func newAzureClusterCR(config ClusterCRsConfig) *capzv1alpha3.AzureCluster {
 		},
 		Spec: capzv1alpha3.AzureClusterSpec{
 			ResourceGroup: config.ClusterID,
+			NetworkSpec: capzv1alpha3.NetworkSpec{
+				APIServerLB: capzv1alpha3.LoadBalancerSpec{
+					Name: fmt.Sprintf("%s-%s", config.ClusterID, "public-lb"),
```

Comment:
Mostly fine, but these names should match with what we deploy in Azure. They are currently defined main `deployment` template here: https://github.com/giantswarm/azure-operator/blob/master/service/controller/azureconfig/handler/deployment/template/main.json#L1100-L1117

Therefore e.g. this LB name should be `<clusterid>-PublicLoadBalancer`.